css/index.css

```diff
@@ -0,0 +1,62 @@
+
+body {
+    background-color: #bab8e9;
+    color: rgb(27, 3, 73);
+    font-size: 1em;
+    font-family: "Open Sans", "Helvetica Neue", sans-serif;
+    margin: 10px;
+    padding: 5px;
+  }
```

Comment:
You don't have to change this if you don't want to, but if you want to change the "leave a message" section so the labels are vertically aligned, you can add this:
```
 label {
    display: inline-block;
    width: 100px;
    text-align: right;
  }
```
The `display` property is quite nuanced, but the basic idea behind `display: inline-block` is to allow setting the width and height and also to allow stacking elements side by side (label + input). So here we're saying we want labels to have a fixed width, the text should be aligned to the right, and we want to align the subsequent element (the input) on the same line.